src/components/field/index.js

```diff
@@ -452,14 +473,11 @@ export default class TextField extends PureComponent {
   inputProps() {
     let store = {};
 
-    for (let key in TextInput.propTypes) {
+    for (let key in this.props) {
```

Comment:
Does that not create a lot of "wrong" unexpected prop-types issues?

Reply:
Adapted the check to not pass any of the react-native-material-textfield props to RN TextInput